### PR TITLE
:information_desk_person: update single time table

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-151.dev.beta.nhschoices.net)

**Comment by ajthomascouk** [comment]
@alex-colbeck thoughts?

**Review by alex-colbeck — DISMISSED** [review]
Looks good to me. Saves loads of space on mobile.

**Comment by beseven** [comment]
@ajthomascouk - can you rebase please?

**Review by beseven — DISMISSED** [review]
Looks ok to me, can you show the team @ajthomascouk through a screenshare/etc?

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-151.dev.beta.nhschoices.net)

File: app/lib/continuousTimeUtils.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ function addTimePadding(parsedTimes, isOneSession) {
     if (time.sessions.length < max) {
       // eslint-disable-next-line no-param-reassign
       time.padding = (max - time.sessions.length);
-    } else if ((time.sessions.length === 1) && (isOneSession === true)) {
+    } else if ((time.sessions.length === 1) || (isOneSession === true)) {
```
Comment (markysoft) [inline app/lib/continuousTimeUtils.js]: This logic seems to be in place to always force the exceptional opening time to use the single session styling, regardless of how many sessions there are.

This leads to bad formatting on the page, i.e. http://profiles-pr-151.dev.beta.nhschoices.net/gp-surgeries/41823

**Review by markysoft — CHANGES_REQUESTED** [review]
Exceptional opening times are forced to use the single session styling regardless of how many sessions giving formatting issues.

File: app/lib/continuousTimeUtils.js (hunk 1 above)
Comment (ajthomascouk) [reply app/lib/continuousTimeUtils.js]: Well spotted. I set it to `||` as it wasn't adding the `one-session` class for "normal" opening hours. I've made that work by setting the flag to true in [`openingTimesMapper.js`](https://github.com/nhsuk/profiles/pull/151/commits/cec41ce6a225501377e05f017653c321d4603818#diff-8e199386d5cabc6984ff1b2c134f9236)

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-151.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-151.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-151.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of profiles succeeded (http://profiles-pr-151.dev.beta.nhschoices.net)

**Review by alex-colbeck — APPROVED** [review]
Great work team 👍

**Review by markysoft — APPROVED** [review]

**Comment by alex-colbeck** [comment]
🔥  good work.

**Pull request merged**